docs/how-to/slice-a-package.md

```diff
@@ -0,0 +1,626 @@
+# Slice a package
+
+This guide provides instructions on the slicing of an Ubuntu package for
+Chisel.
+
+A package slice is represented via a
+{ref}`Slice Definitions File<slice_definitions_ref>` (aka SDF), and the creation
+of said slice definitions is a result of the **inspection** and **design
+process** depicted below.
+
+As an example, this guide will focus on **slicing the `openssl` package from
+Ubuntu 24.10**.
+
+(create_sdf_inspect_pkg)=
+
+## Inspect the package
+
+Your first task is to **understand the package** to be sliced.
+
+Packages are primarily composed of files for installation and may reference
+additional packages as dependencies such as libraries or other resources.
+
+During the slicing process, we identify subsets of files in a package which can
+be installed alone to accomplish one or more tasks. Consequently, the
+dependencies required for this slice may be a subset of the dependencies
+indicated by the package.
+
+1. [ ] **Is it an Ubuntu package?**
+
+    Chisel only supports packages from the Ubuntu (and Ubuntu Pro) archives.
+
+2. [ ] [**Get the package dependencies**]{#create_sdf_inspect_pkg_deps}
+
+    You can run `apt depends openssl` or use the [Ubuntu Packages Search] website.
+
+3. [ ] [**Inspect the files the package provides**]{#create_sdf_inspect_pkg_files}
+
+    You can run `apt download openssl` and then `dpkg -c openssl*.deb` to check
```

Comment:
What do you think about having the commands in a code block? Would be easier to copy and spot.

Reply:
TBH i'd just leave it like this, cause it's just one way of doing it. "You **can**...", meaning that there are other ways, and since this is a guide, some steps are purely suggestive.